styles.css

```diff
@@ -1,4 +1,46 @@
+* {
+  border: dashed 1px red;
+}
+
 body { font-family: 'Roboto', sans-serif; }
 
-.section__title,
+.main-section__title,
 .header__logo { font-family: 'PT Serif', serif; }
+
+.wrapper {
+  width: 80%;
+  margin: 0 auto;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.header__logo { padding: 40px 0; }
+
+.nav {
+  display: flex;
+  align-items: center;
+}
+
+.nav__link { 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/html_2.0/pullRequest?prid=6300592)